modules/admin-bar/module.php

```diff
@@ -0,0 +1,132 @@
+<?php
+namespace Elementor\Modules\AdminBar;
+
+use Elementor\Plugin;
+use Elementor\Core\Base\Document;
+use Elementor\Core\Base\Module as BaseModule;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+	/**
+	 * @var Document[]
+	 */
+	private $documents = [];
+
+	/**
+	 * @return bool
+	 */
+	public static function is_active() {
```

Comment:
Where is this method used?

Reply:
The Module Manager calls this method before it registers the module.